app/v1/dist/index.html

```diff
@@ -225,6 +239,20 @@
           gui = new dat.GUI({ autoPlace: false });
 console.log(`QQ/gui/domElement: ${gui.domElement}`);
           viewerElement.appendChild(gui.domElement);
+          
+          //Add a download button to the page it TODO: this process will need to updated for downloading other file types like .pdf
```

Comment:
How about we stick the downloadButton adding code in a little function, so that it's clear where the boundary of the code is?